source/getting-started/quick-start.md

```diff
@@ -74,6 +74,8 @@ Let's create a new template using the `ember generate` command.
 ember generate template application
 ```
 
+Ember CLI will notice that there's already a template there and ask if you want to overwrite it. Enter `y` to continue.
```

Comment:
if there's already a template there then why are we generating it?  Is this a leftover from the few releases when Ember CLI was not generating the application template?  If so we should just skip the generate command.